apps/web/src/design-system/button/Button.styles.ts

```diff
@@ -45,6 +45,10 @@ export default createStyles(
         '&&:hover': {
           backgroundSize: '100%',
         },
+        '&:focus': {
+          outlineOffset: 0,
+          outline: '2px solid rgb(153,200,255)',
+        },
```

Comment:
By default the tour library that we do use supports the keyboard navigation, this line is a style for the focused button. I've made it look the same styles as the mantine has for the unstyled button (skip tour button). I couldn't find in the design system how it supposed to look, but I know that we will change it in the near future as Nikolay wanted to improve the components.